.github/workflows/linux-pipeline.yml

```diff
@@ -167,22 +162,41 @@ jobs:
         rm -rf src/third_party/swiftshader/tests/regres/testlists
         rm -rf src/electron
         rm -rf src/third_party/electron_node/deps/openssl
-        rm -rf src/third_party/electron_node/deps/v8
     - name: Compress Src Directory
       if: steps.check-cache.outputs.cache_exists == 'false'
       run: |
         echo "Uncompressed src size: $(du -sh src | cut -f1 -d' ')"
         tar -cvf $DEPSHASH.tar src
         echo "Compressed src to $(du -sh $DEPSHASH.tar | cut -f1 -d' ')"
-    - name: Upload Compressed Src Cache to Azure
+    - name: Move src folder to cross-OS portal
+      if: steps.check-cache.outputs.cache_exists == 'false'
+      run: |
+        cp ./$DEPSHASH.tar /mnt/cross-instance-cache/
+        sudo mkdir -p /var/portal
+        sudo chown -R $(id -u):$(id -g) /var/portal
+        mv ./src /var/portal
+    - name: Persist Src Cache
       if: steps.check-cache.outputs.cache_exists == 'false'
       run: |
-        az storage blob upload \
-          --account-name $AZURE_STORAGE_ACCOUNT \
-          --account-key $AZURE_STORAGE_KEY \
-          --container-name $AZURE_STORAGE_CONTAINER_NAME \
-          --file $DEPSHASH.tar \
-          --name $DEPSHASH
+        backup_cache_path=/var/portal
+        final_cache_path=/mnt/cross-instance-cache/$DEPSHASH.tar
+        echo "Using cache key: $DEPSHASH"
+        echo "Checking path: $final_cache_path"
+        if [ ! -f "$final_cache_path" ]; then
+          echo "Cache key not found, storing tarball"
+          tmp_container=/mnt/cross-instance-cache/tmp/${{ github.sha }}
+          tmp_cache_path=$tmp_container/$DEPSHASH.tar
+          mkdir -p $tmp_container
+          if [ -f "$backup_cache_path" ]; then
```

Comment:
What's this `backup_cache_path` logic?

Reply:
It's a `/var/portal` fallback, the backup cache points to `/var/portal` in case the /mnt/cross-instance path wasn't found. Honestly though, I had put that in while trying to make `/mnt/cross-instance` work; I don't think we need this and can just rely on `/mnt/cross-instance`, I'll remove it 👍